stable/traefik/values.yaml

```diff
@@ -3,6 +3,7 @@ image: traefik
 imageTag: 1.3.4
 serviceType: LoadBalancer
 loadBalancerIP:
+#loadBalancerSourceRanges: []
```

Comment:
Of course. I can rewrite the PR if you wish. I was looking at how other charts were doing this same thing (i.e. prometheus) and they seem to have a flat structure. 

I'll commit the changes.